### ISSUE TITLE
Type of parameter "self" is unknown (reportUnknownParameterType) when subclassing typing.Protocol

### ISSUE THREAD

**Issue by helafumpek** [microsoft/pylance-release#458]
## Environment data

-   Language Server version: 2020.9.8
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): Anaconda Python 3.8.5

## Expected behaviour

The "self" parameter should have the type of class to which its method belongs (as it is in other cases).

## Actual behaviour

```python
from typing import Protocol

class Foo(Protocol):
    def bar(self) -> None:
        ...
```

reports

```
(parameter) self: Unknown
"self" is not accessed Pylance
Type of parameter "self" is unknown Pylance (reportUnknownParameterType)
```

**Comment by jakebailey** [comment]
@erictraut Seems like a bug for `self` to be marked as unknown in this context (or any context, really...)

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.10.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020100-7-october-2020
